desktop-widgets/diveplanner.cpp

```diff
@@ -232,10 +244,42 @@ void DivePlannerWidget::heightChanged(const int height)
 	plannerModel->setSurfacePressure(pressure);
 }
 
-void DivePlannerWidget::salinityChanged(const double salinity)
+void DivePlannerWidget::waterTypeUpdateTexts()
 {
-	/* Salinity is expressed in weight in grams per 10l */
-	plannerModel->setSalinity(lrint(10000 * salinity));
+	double density;
+	for (int i = 0; i < 3; i++) {
+		if (ui.waterType->itemData(i) != QVariant::Invalid) {
+			QString densityText = ui.waterType->itemText(i).split("(")[0].trimmed();
+			density = ui.waterType->itemData(i).toDouble();
+			densityText.append(QString(" (%1kg/l)").arg(QString::number(density, 'f', 2)));
```

Comment:
Any reason you don't use `QString::arg(double, ...)` here?

Reply:
Yes, I want the output to be fixed with two digits. I don't know how to do it without QString::number.